pkg/cmd/bazci/bazci.go

```diff
@@ -201,7 +201,7 @@ func getBuildInfo(args parsedArgs) (buildInfo, error) {
 		switch targetKind {
 		case "cmake":
 			ret.cmakeTargets = append(ret.cmakeTargets, fullTarget)
-		case "genrule":
+		case "genrule", "batch_gen":
```

Comment:
@rickystewart I'd like you to sign off on this one line. I created a rule because I needed to access info inside the `GoSource` provider and I couldn't figure out how to do that in a `genrule`.